models/src/test/scala/coop/rchain/models/HashMSpec.scala

```diff
@@ -37,7 +37,9 @@ class HashMSpec extends FlatSpec with PropertyChecks with Matchers {
   sameResultAsReference[ESet]
   sameResultAsReference[ETuple]
   sameResultAsReference[EVar]
+  sameResultAsReference[GUnforgeable]
   sameResultAsReference[GPrivate]
+  sameResultAsReference[GDeployerAuth]
```

Comment:
This one seems redundant, please confirm

Reply:
Hmm, why does it seem redundant?

Reply:
The test for GUnforgrable should explore these as well. No big deal though :)